### ISSUE TITLE
Partials aren't found

### ISSUE THREAD

**Issue by sidabs** [ConnorWiseman/koa-hbs-renderer#8]
Error I'm getting: `The partial footer could not be found`.  My setup is below.  Am I missing a step to register my unique partials?  I thought that was covered by the `options.paths.partials` setting passed to the `renderer`?

I've used this in my main `/lib/app.js`:
```js
  app.use(renderer({
    cacheExpires: 60,
    contentTag: 'content',
    defaultLayout: '../../views/layouts/main',
    environment: 'development',
    extension: '.hbs',
    paths: {
      views: path.join(process.env.PWD, '/views'),
      layouts: path.join(process.env.PWD, '/views/layouts'),
      partials: path.join(process.env.PWD, '/views/partials'),
    },
  }));
```
My `/views/layouts/main.hbs` file is the following:
```html
<!DOCTYPE html>
<html>
  <head>
    <title>{{title}}</title>
  </head>
  <body>
    <div class="header-wrapper">
      <h1>({{title}})</h1>
    </div>
    {{{content}}}
    {{> footer}}
  </body>
</html>
```
`/views/partials/footer.hbs` partial file is:
```html
<span>This is the footer</span>
```
Router action is:
```js
  router.get('/', async function (ctx) {
    await ctx.render('peprojects', {
      title: 'useful',
    });
  });
```

**Comment by ConnorWiseman** [comment]
Thanks for reporting this! I was able to reproduce the problem and narrowed the cause down to a typo on line 88. It should be fixed as of v1.0.1.

**Comment by sidabs** [comment]
@ConnorWiseman thanks for the quick fix!  Works like a charm now!
